egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1i_dp.sh

```diff
@@ -243,6 +244,7 @@ if [ $stage -le 16 ]; then
     --egs.chunk-left-context $chunk_left_context \
     --egs.chunk-right-context $chunk_right_context \
     --trainer.dropout-schedule $dropout_schedule \
+    --trainer.dropout-per-frame $dropout_per_frame \
```

Comment:
as Vimal says, please remove this from the training code... does not need to be there.